onnx_tf/handlers/backend/pow.py

```diff
@@ -4,16 +4,75 @@
 from onnx_tf.handlers.handler import onnx_op
 from onnx_tf.handlers.handler import tf_func
 from .math_mixin import BasicMathMixin
+from onnx_tf.common import sys_config
 
 
 @onnx_op("Pow")
 @tf_func(tf.pow)
 class Pow(BasicMathMixin, BackendHandler):
 
+  @classmethod
+  def _common(cls, node, **kwargs):
+
+    def is_safe_cast(from_dtype, to_dtype):
+      safe_cast_map = {
+          tf.bfloat16: [tf.float32, tf.float64],
+          tf.float16: [tf.float32, tf.float64],
+          tf.float32: [tf.float64],
+          tf.int8: [tf.float16, tf.float32, tf.float64, tf.int32, tf.int64],
+          tf.int16: [tf.float16, tf.float32, tf.float64, tf.int32, tf.int64],
+          tf.int32: [tf.float32, tf.float64, tf.int64],
+          tf.int64: [tf.float64],
+          tf.uint8: [tf.float16, tf.float32, tf.float64, tf.int32, tf.int64],
+          tf.uint16: [tf.float32, tf.float64, tf.int32, tf.int64],
+          tf.uint32: [tf.float64, tf.int64]
+      }
+      return to_dtype in safe_cast_map[from_dtype]
+
```

Comment:
@chinhuang007 may be move this "is_safe_cast" function to onnx_tf.common so other handler can use this function too.

Reply:
Good idea. Moved "is_safe_cast" to common.data_type.py and added complex types. Please review. Thanks!